app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -29,6 +29,7 @@ const Note = ({ note, setSelectedNoteIds, setShowDeleteAlert }) => (
     <div className="mt-3 flex justify-between">
       <Tag
         className="neeto-ui-bg-gray-200"
+        // Change this line to show selected tags , keep this as default for now.
```

Comment:
If the comment is meant as a TODO, then prefix with a `TODO: ` keyword in the comment.